core/controllers/creator_dashboard_test.py

```diff
@@ -644,6 +679,77 @@ def test_can_see_subscribers(self):
         response = self.get_json(feconf.CREATOR_DASHBOARD_DATA_URL)
         self.assertEqual(len(response['subscribers_list']), 0)
 
+    def test_can_create_new_topic(self):
```

Comment:
I don't see why this test goes into this file...

Reply:
This is required to test the the if clause [here](https://github.com/oppia/oppia/blob/develop/core/controllers/creator_dashboard.py#L338) to see if `topic_summary_dicts` gets updated. I have updated its name for clearer understanding